zebra-scan/src/storage/db.rs

```diff
@@ -46,11 +46,11 @@ impl Storage {
     /// If there is no existing database, creates a new database on disk.
     ///
     /// New keys in `config` are not inserted into the database.
-    pub(crate) fn new_db(config: &Config, network: Network) -> Self {
+    pub(crate) fn new_db(config: &Config, network: Network, read_only: bool) -> Self {
         Self::new_with_debug(
             config, network,
             // TODO: make format upgrades work with any database, then change this to `false`
```

Comment:
This TODO is now ambiguous

```suggestion
            // TODO: make format upgrades work with any database, then change debug_skip_format_upgrades to `false`
```